### DISCUSSION TITLE
RTL should be centered in Other languages > Target string

### DISCUSSION THREAD

**Discussion by NamelessGO** [WeblateOrg/weblate#11086] [Ideas] [answered]
Reasons: 
- Users will see RTL and think the translation is missing
- Users expect the Target string to be centered even for RTL

![image](https://github.com/WeblateOrg/weblate/assets/66227691/17caf751-8676-4694-b240-dae71fac2205)

Searched issues and Discussions for "RTL", "Target string", and "others languages", but couldn't find any post about it, if it is duplicated please post the link to bump :+1:

**Comment by jermanuts** [comment, marked as answer]
Agree, but I doubt that it will _work_ without a whole change to the CSS (I might be wrong). Currently, for the Arabic and other RTL languages it has a dir set to "rtl" and other languages to "ltr". The example you provided doesn't look right because it is a one word, when it is a whole sentence it will look weird with dir set to "ltr" for RTL languages.

So leave it as it is.
